-------------------------------------------------------------------------------
Test set: com.app.dao.UserEntityDaoTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.681 s <<< FAILURE! - in com.app.dao.UserEntityDaoTest
testAddUsers  Time elapsed: 0.3 s  <<< ERROR!
org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; constraint [secure_users.UK_m0hkyp9citwjqb5qy5n2qp2qo]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement
	at com.app.dao.UserEntityDaoTest.testAddUsers(UserEntityDaoTest.java:37)
Caused by: org.hibernate.exception.ConstraintViolationException: could not execute statement
	at com.app.dao.UserEntityDaoTest.testAddUsers(UserEntityDaoTest.java:37)
Caused by: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'a1@gmail.com' for key 'secure_users.UK_m0hkyp9citwjqb5qy5n2qp2qo'
	at com.app.dao.UserEntityDaoTest.testAddUsers(UserEntityDaoTest.java:37)

